src/Aspire.Hosting.MongoDB/MongoDBBuilderExtensions.cs

```diff
@@ -12,6 +12,7 @@ namespace Aspire.Hosting;
 /// </summary>
 public static class MongoDBBuilderExtensions
 {
+    private const string MongoDBContainerDataDirectory = "/var/mongodb/data";
```

Comment:
Please revert the changes to this file.

1. We shouldn't be making unnecessary/unrelated changes in a PR. Keep each PR to a single change.
2. This isn't correct. From https://stackoverflow.com/questions/35400740/how-to-set-docker-mongo-data-volume it needs to stay `/data/db`.

Have you tested if the data volumes still work?

Reply:
Ready. Canceled.
Sorry. This is because I want to do a little more. I fall for this all the time.
Although I understand perfectly well. What one PR needs to do to solve one problem.

Reply:
> Have you tested if the data volumes still work?

Honestly, no. I just saw that the tests were working. I didn’t run it separately to check data volumes